content/news/XXXX-XX-XX-mixxx-2-4-1-released.md

```diff
@@ -0,0 +1,155 @@
+title: Mixxx 2.4.1 released
+authors: Evelynne Veys
+tags: 2.4, 2.4.1, release announcement
+comments: yes
+status: draft
+
+Dear Mixxx-ers,
+
+The Mixxx-developingteam started the year with good intentions and therefore imposed itself some deadlines. That is why only two months after version 2.4, 
+
+the next release: version 2.4.1 can be offered. This isn't just a service release fixing some minor bugs and problems, version 2.4.1 contains some nice
+
+features. Hiding them longer for you could possibly been seen as a crime.
```

Comment:
There are no new features in 2.4.1. What are you referring to?

Reply:
The long press sliding change of the sync knob, addings to controller mappings,
controllermapping updates..
You should be more proud of yourself/yourselves, it 's not just solving minor bugs.

Reply:
you could see 95% of Mixxx as a kind of fix, but for me those are all features.

Reply:
I disagree, the 'feature' defintion I posted is pretty clear IMHO.
I simply find it misleading to announce new features for a bugfix release.

Please try to reword that paragraph.

Reply:
What did you write to me about what I should do when you 'disagree' ? :-)
Let me think

Reply:
For me it is just the other way around a missing feature is a bug. A feature is clearly  somemething that may remove or alter a user workflow. So you see, we need a definition for the gray area.

Fortumtly we have one: A point release is a bugfix release and 2.5.0 will be a feature release. 

I must admit that I have merged some PRs that we may consider as features. Because they where of a low risk and limited scope I have just merged them.

Conclusion: I agree with @ronso0 that we must not advertise this as a feature release to not scare away users because of regression assumptions for features.

On the other hand we can be proud and enthusiastic about this release. Let me have a look if I find a wording that covers that.

Reply:
I already changed it